lib/common/models/concerns/cache_aside.rb

```diff
@@ -42,7 +42,11 @@ def do_cached_with(key:)
 
     def cache(key, response)
       set_attributes(key, response)
-      save
+      if response.respond_to?(:record_ttl)
```

Comment:
again, lets not change this mixin up until we're certain that we need different mappings from Class -> Cache. I'm not entirely sure we need this, and if we do, I'm still largely in favor of introducing an STI type hierarchy where you create a new class derived from the base class and override these methods.